projects/CryptooTech/index.js

```diff
@@ -0,0 +1,14 @@
+const { sumTokens2, nullAddress } = require('../helper/unwrapLPs')
+
+const contract = "0x4a3720A09a90A82170779c972832eD04542deeAC"
```

Comment:
is this contract deployed/ right contract? 

https://basescan.org/address/0x4a3720A09a90A82170779c972832eD04542deeAC

Reply:
yes its right contract, but deployed in arbitrum.